cmd/suseconnect.go

```diff
@@ -94,6 +115,21 @@ func main() {
 	} else if deregister {
 		err := connect.Deregister()
 		exitOnError(err)
+	} else {
+		if instanceDataFile != "" && connect.URLDefault() {
```

Comment:
why is this nested and not just another `else if`?

Reply:
> why is this nested and not just another `else if`?

I'm trying to follow the flow of the Ruby version:
https://github.com/SUSE/connect/blob/master/lib/suse/connect/cli.rb#L38-L51

Reply:
hmm... I guess that's fine then.

Reply:
I agree it's not nice. Maybe after we have parity of functionality we can come back and improve the whole cli option processing.